.github/workflows/pylint.yaml

```diff
@@ -0,0 +1,19 @@
+name: main-workflow 
+on: [push]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install pylint
+        pip install -r requirements.txt
+    - name: Lint with pylint
+      run: |
+        pylint main.py duty.py --disable=E1136 --exit-zero
```

Comment:
Change this to `pylint . --recursive=y --ignore-paths venv --disable=E1136 --exit-zero`. 
This will add more flexibility in the future.